cmd/jaeger/config-remote-storage.yaml

```diff
@@ -60,4 +60,4 @@ processors:
 
 exporters:
   jaeger_storage_exporter:
-    trace_storage: some-storage
+    trace_storage: some-storage
```

Comment:
you need grpc storage config to support two different ports, for reading and writing, since in the other binary you cannot run read an write handlers on the same port